server/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN npm install --global nodemon
 # For npm@5 or later, copy package-lock.json as well
 COPY package.json .
 RUN npm install
+RUN npm install nodemailer --save
```

Comment:
Remove this line. When docker starts building the image, it runs 'npm install' and it installs all the dependencies specified in package.json. You do not need to install it "manually" here. "npm install nodemailer --save" will install and save the dependency into the package.json file. This is useful when you develop in your local env, but it's not needed here.

Reply:
Ok, sorry. Done